packages/amplication-server/src/core/analytics/types.ts

```diff
@@ -0,0 +1,19 @@
+export enum EnumTimeFrame {
+  LastMonth = "LAST_MONTH",
+  LastThreeMonths = "LAST_THREE_MONTHS",
+  LastYear = "LAST_YEAR",
+  Custom = "CUSTOM",
+}
+
+export type TimeFrame = {
```

Comment:
I think it's confusing that the type `TimeFrame` has a property `timeFrame`

Reply:
I need to remove the time frame enum and type